source/connection-troubleshooting.txt

```diff
@@ -269,8 +269,9 @@ Monitor Thread Exception
 
    INFO: Exception in monitor thread while connecting to server ssc-cluster-01-shard-00-02.9cbnp.mongodb.net:27017
 
-To resolve this error, you must install certificates as described under
-:ref:`Security Certificate Errors <java-connection-certificate>`.
+This log line indicates that the monitoring thread that continuously
+checks the status of each node in the replica set failed to contact one of the nodes.
```

Comment:
It's either each replica set member _or_ each mongos server, depending on deployment type.